pkg/sparrow/run.go

```diff
@@ -77,6 +80,7 @@ func New(cfg *config.Config) *Sparrow {
 		cErr:       make(chan error, 1),
 		cDone:      make(chan struct{}, 1),
 		shutOnce:   sync.Once{},
+		checker:    healthz.New(cfg.Api),
```

Comment:
Maybe this should just pass the port instead of the whole api config object. We should ensure in some way, that sparrow always checks against localhost:<apiport> for its healthz.

Reply:
To clarify on this: my worry is, that if a user configures some dns name as their api address (whatever the reason for that), we would use that dns name to check the local endpoints. If we were in kubernetes and used the service name for that, the healthz requests would be sent to the service, which would forward it to one of the **already existing** 'old' sparrows. Since they are already alive, the healthcheck would be green, causing kubernetes to flag the new sparrow as healthy, even if it hasn't yet completed it's startup or is faulty

Reply:
I've added a `formatAddress` utility function that handles various api address formats. 

For more information, look at the unit tests I've provided: https://github.com/caas-team/sparrow/commit/f8cef21e23467e92a58ecd7067c811ee0c06e5d1#diff-d0b860d237946173d5459e969e0e7789b1ef1c55856deb36608a9900856bf798R115